hudi-utilities/src/main/java/org/apache/hudi/utilities/deser/KafkaAvroSchemaDeserializer.java

```diff
@@ -70,13 +69,12 @@ public void configure(Map<String, ?> configs, boolean isKey) {
    */
   @Override
   protected Object deserialize(
-      boolean includeSchemaAndVersion,
       String topic,
       Boolean isKey,
       byte[] payload,
       Schema readerSchema)
       throws SerializationException {
-    return super.deserialize(includeSchemaAndVersion, topic, isKey, payload, sourceSchema);
+    return super.deserialize(topic, isKey, payload, sourceSchema);
```

Comment:
Did we ever not include schema and version?

Reply:
This option is no longer available in the confluent library